### PR TITLE
Add sideEffects field to applicationinsights-shims package.json #1384

### PR DESCRIPTION
**Pull request by MSNev**

### CODE REVIEW

File: tools/shims/package.json [hunk 1]
```diff
@@ -27,7 +27,10 @@
         "url": "https://github.com/microsoft/ApplicationInsights-JS/tree/master/tools/shims"
     },
     "license": "MIT",
-    "sideEffects": false,
+    "sideEffects": [
```
Comment (markwolff) [inline tools/shims/package.json:30]: Does this affect the sideEffects of consumers of this package?

Comment (MSNev) [reply tools/shims/package.json:30]: Yes, specifically [webpack tree shaking](https://webpack.js.org/guides/tree-shaking/),

Comment (markwolff) [reply tools/shims/package.json:30]: Do each of their sideEffects properties need to be updated as well?

Comment (MSNev) [reply tools/shims/package.json:30]: No, only on the the published module that actually have side effects that are not exported as variables. In this specific case the side effect is the setting / exposing of the global __extends() and __assign() functions. Just by loading the module.

**Review by markwolff — APPROVED** [review]

**Pull request merged**